plugins/builds/create.js

```diff
@@ -12,8 +12,8 @@ module.exports = () => ({
         notes: 'Create and start a specific build',
         tags: ['api', 'builds'],
         auth: {
-            strategies: ['token', 'session'],
-            scope: ['user']
+            strategies: ['token'],
+            scope: ['user', '!guest']
```

Comment:
I have a question. From my understanding, this `scope` array is an OR. For example if `scope: ['user', 'build']` means either a build or a user is good to access.
In this case, don't we want an `AND`? It's a user AND it's not a guest (since guest users have both `user` and `guest` scope). Maybe I missed something. Could you explain?

Reply:
So for HapiJS, scope works like this:
 - If it starts with `+` that scope is required
 - If it starts with `!` that scope is not allowed
 - Otherwise it's an OR

More info [here](https://hapijs.com/api#-routeoptionsauthaccessscope).